### DISCUSSION TITLE
Remove Milestones and releases candidate releases versions from SDKMAN

### DISCUSSION THREAD

**Discussion by sdelamo** [micronaut-projects/micronaut-starter#1987] [General]
I think we should we release Milestones and releases candidate versions from SDKMAN

![CleanShot 2023-07-31 at 13 16 11](https://github.com/micronaut-projects/micronaut-starter/assets/864788/430af21f-7431-4bd1-88db-7a075c959705)

@micronaut-projects/core-developers thoughts?

**Comment by wetted** [comment]
I agree with removing them. Last time I used SDKMAN I wondered why they were all there, especially the old ones.

**Comment by timyates** [comment]
I agree 👍

**Comment by melix** [comment]
Fine by me too
